ports/ez/port.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": 3,
+  "name": "ez.zip",
+  "items": [
+    "E.Z.sh",
+    "ez"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "E.Z",
+    "porter": [
+      "Fraxinus88"
+    ],
+    "desc": "E.Z is a simple hardcore platformer game the goal being get across to the other side of each of the 100 levels while each death throws you back to the previous levels.\n\nWith a semi-random level sequence, level variability and quick level swapping, E.Z is hard to get used to. It will bring a challenge to your skill, reaction, and adaptability.\n\nNote: Text may be unreadable on smaller devices but this does not affect gameplay!",
```

Comment:
Same as on the other game. You can't work with \n\ on the on device desctiption. Keep it short on device and elabortate on the desc_md. 

Same with the instructions

Reply:
Okies edited kindly review